redisvl/query/query.py

```diff
@@ -21,6 +21,86 @@ def params(self) -> Dict[str, Any]:
         raise NotImplementedError
 
 
+class CountQuery(BaseQuery):
```

Comment:
Should this also have return fields? I guess you could just do a FilterQuery and use ``len()``

Reply:
The idea of count query is that it returns `no_content()` thus is a more efficient query than doing a filter query with len